spec/models/elastic_sayt_suggestion_spec.rb

```diff
@@ -1,29 +1,47 @@
 require 'spec_helper'
 
-# Temporarily disabling these specs during ES56 upgrade
-# https://cm-jira.usa.gov/browse/SRCH-821
 describe ElasticSaytSuggestion do
   fixtures :affiliates
   let(:affiliate) { affiliates(:basic_affiliate) }
+  let(:search_params) do
+    {
+      q: 'suggest',
+      affiliate_id: affiliate.id,
+      size: 1,
+      offset: 0,
+      language: affiliate.indexing_locale
+    }
+  end
+  let(:search) { ElasticSaytSuggestion.search_for(search_params) }
 
   before do
     ElasticSaytSuggestion.recreate_index
     affiliate.sayt_suggestions.delete_all
     affiliate.locale = 'en'
   end
 
-  pending ".search_for" do
-    describe "results structure" do
+  describe '.search_for' do
+    describe 'results structure' do
       context 'when there are results' do
+        let(:search_params) do
+          {
+            q: 'suggest',
+            affiliate_id: affiliate.id,
+            size: 1,
+            offset: 1,
+            language: affiliate.indexing_locale
+          }
+        end
+
```

Comment:
That was my initial plan, but the inclusion of `offset: 1` in the `.search_for` specs makes the params slightly different from what I would expect to be the default (`offset: 0`). I was trying to avoid refactoring the specs to use a different offset. 

That said, it clearly bugs us both that these params are not DRY, so I'm going to suck it up and do that refactoring. So there will be some additional spec changes coming beyond the `search_params` reuse.